RadixWallet/Features/InteractionReview/Helpers/DisplayMode.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+extension InteractionReview {
+	enum DisplayMode: Sendable, Hashable {
+		case detailed
+		case raw(String)
```

Comment:
can we use something more structured than `String`? Don't we have some `UnvalidatedManifest` type or something? Maybe we can create a new type or some typealias at least.

I would also put a one line doc on both cases and probably give `raw` a label: `case raw(manifest: ...)`.

Reply:
We cannot use `UnvalidatedManifest` since in `TransactionReview` we are dealing with a `TransactionManifest` (to which we append the fee & guarantees configurations done at this view), while at `PreAuthroizationReview` we deal with a `SubintentManifest`.

I will add the `raw` label, but I think adding a type around the `String` is an overkill, since this is enum is just internally to display the `RawManifestView`.